src/stratis_cli/_actions/_logical.py

```diff
@@ -158,6 +161,12 @@ def filesystem_used(props):
                 props, "Used", lambda x: str(Range(x)), TABLE_FAILURE_STRING
             )
 
+        format_uuid = (
+            lambda mo_uuid: mo_uuid
+            if namespace.unhyphenated_uuids
+            else str(uuid.UUID(mo_uuid))
```

Comment:
This is actually slightly different from what I had in mind, I didn't notice before. What I want is for ```format_uuid``` to be assigned one of two functions depending on the value of ```namespace.unhyphenated_uuids```. This is just one anonymous function. For two functions, you need:

```format_uuid = lambda mo_uuid: mo_uuid if namespace.unhyphenated_uuids else lambda mo_uuid: str(uuid.UUID(mo_uuid))```

Reply:
Actually I'ld like to revise my request here:

1. Define a function that does the uuid transformation in ```_formatting.py```. Call it say, ```to_hyphenated```.
2. Then here: ```format_uuid = lambda mo_uuid: mo_uuid if namespace.unhyphenated_uuids else to_hyphenated```.

I think that's best.

Reply:
@mulkieran I see, because my version still causes the function to evaluate the "if else" statement on every call to `format_uuid`, whereas with your solution, the "if else" is only evaluated once, correct? I was confused by the syntax but I see it now.

Reply:
@lleshchi In Python ```lambda <id> : <expr>``` is always interpreted to be a single function, even if ```<expr>``` has ```lambda``` in it.

```
>>> thing = True
>>> func = lambda x: x if thing else lambda x: x + 1
>>> func(1)
1
>>> thing = False
>>> func(1)
<function <lambda>.<locals>.<lambda> at 0x7f8ef62948b0>
```

Not what was intended. To persuade Python to parse properly, we need some parens:

```
>>> thing = False
>>> new_func = (lambda x: x) if thing else (lambda x: x + 1)
>>> new_func(1)
2
```
Again:

```
>>> thing = True
>>> new_func = (lambda x: x) if thing else (lambda x: x + 1)
>>> new_func(1)
1
```

Note that the value of ```new_func``` is either ```lambda x: x``` or ```lambda x: x + 1``` once the assignment expression is evaluated.

Note that this means that, even with the separate function defined in ```_formatting.py``` you must still use the parens to override Python's default parsing for lambda-expressions.